.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,76 @@
+
+  - type: input
+    id: group
+    attributes:
+      label: Group
+      description: The lab, faculty member, PI, team, org, or group this project is primarily for.
+      placeholder: e.g. Greene Lab
+
+  - type: textarea
+    id: contact
+    attributes:
+      label: Contact info
+      description: Contact and name info for key parties for the project.
+      placeholder: |
+        E.g.:
+        Casey Greene, PI, casey@greenelab.com, @caseygreene on slack
+        John Doe, project lead, john.doe@fake.com, @johndoe on slack
+
+  - type: textarea
+    id: repos
+    attributes:
+      label: Links to code
+      description: |
+        Link to any existing code you have on e.g. GitHub, GitLab, Bitbucket, etc., and provide us the appropriate permissions to view it ([invite all team members](https://cu-dbmi.github.io/set-website/about#the-team)). This will help us to proactively answer questions, estimate time, find solutions, contribute fixes, etc.
+      placeholder: |
+        E.g.:
+        https://github.com/greenelab/greenelab.com
+
+  - type: textarea
+    id: workflow
+    attributes:
+      label: Workflow
+      description: |
+        Describe your software workflow and practices. E.g.:
+        - Do you use git and GitHub for tracking code?
+        - What operating system do you use?
+        - What programming languages, package managers, and notable libraries do you use?
+        - What IDE/editor or other tools do you use?
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Work description
+      description: |
+        Describe the specific support you need from SET. E.g.:
+        - What kind of [project work](https://cu-dbmi.github.io/set-website/about#what-we-do) needs to be done?
+        - What platforms do you need to target?
+        - What is the "minimum viable product"? What are the different phases of the project?
+        - Are there HIPAA concerns?
+        - Is there a long term maintenance plan?
+
+  - type: textarea
+    id: timeline
+    attributes:
+      label: Timeline
+      description: |
+        Describe all of your hard and soft deadlines for specific parts of the support.
+
+  - type: textarea
+    id: funding
+    attributes:
+      label: Funding
+      description: |
+        Describe your funding situation in detail. The department can cover SET's costs up to a point (e.g. for small projects and tech support), but you may be asked to cover some or all of the costs with your own grant (or other) funds, especially for larger and more long-term projects.
```

Comment:
I see the value of asking for this, since it'd be nice to record this along with the request, but I wonder if it's our concern. Further, I'm a little concerned that this might not be good to make public, e.g. if they're contending with other groups for funding or have a grant that's  not yet approved and don't want to "show their hand". In any case, I think we should get some input from the higher-ups on this.

Reply:
The reason I included it is because I think we or Audrey will always need to know this info eventually. This question could go in the dept's "grant proposal/intake" form where the responses are private, but someone could still need our support without having a grant. 

But I could definitely see this being info they want to keep private, in which case it will just have to go to Audrey directly (as an email or yet another form). Let's see what she thinks.

Reply:
https://github.com/CU-DBMI/set-intake/pull/1/commits/675c3d4b529af21721ab761c3169509120c71eab